src/comm.h

```diff
@@ -23,6 +17,13 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+/**
+ * @file comm.h
```

Comment:
Adding @file sections to these .h files includes them in the doxygen run so now the doc check is failing
because doxygen expects these headers to be documented.

I think we should check if the doxygen config can be adjusted to allow function declarations to be undocumented.